kumascript/macros/ListSubpagesForSidebar.ejs

```diff
@@ -34,111 +30,90 @@ const overview = mdn.localString({
     'ko': '개요',
     'ru': 'Обзор',
     'zh-CN': '概述',
+    'zh-TW': '概述'
 });
 
-var parent = await wiki.getPage(path);
-var pages = includeParent ? [{
-    ...parent,
-    title: overview
-}, ...parent.subpages] : [...parent.subpages];
-var containsTag = page.hasTag;
-var htmlEscape = kuma.htmlEscape;
-
-var output = '';
-var openTag = '<ol>';
-var closeTag = '</ol>';
-var code    = !$1 ? '<code>'  : '';
-var endcode = !$1 ? '</code>' : '';
-
-// Strings
-var commonl10n = web.getJSONData('L10n-Common');
-var text = {
-    'translate': mdn.getLocalString(commonl10n, '[Translate]'),
-    'title': mdn.getLocalString(commonl10n, 'TranslationCTA'),
-};
-
-var badges = {
-  ExperimentalBadge : await template("ExperimentalBadge"),
-  NonStandardBadge : await template("NonStandardBadge"),
-  DeprecatedBadge : await template("DeprecatedBadge"),
-  ObsoleteBadge : await template("ObsoleteBadge"),
+const parent = await wiki.getPage(path);
+const pages = parent.subpages;
+
+// trim english title for sorting
```

Comment:
For readability, let's start all comments with an uppercase letter, and end them with a full-stop.

```suggestion
// Trim english title for sorting.
```